src/d3a/models/const.py

```diff
@@ -104,6 +104,13 @@ class IAASettings:
         # Option 2 stands for double sided pay as bid market
         # Option 3 stands for double sided pay as clear market
         MARKET_TYPE = 1
+        # PRICING_SCHEME
```

Comment:
If there are more settings in the following subtasks (like for multiple processes), I would suggest to create yet another level of nesting, in order to distinguish it in a better way.

Reply:
Ok! Good Idea!